www/home/form.py

```diff
@@ -40,16 +56,21 @@ def format_results(self, database_data):
                 formatted_courses["CourseName"] = course.CourseName
                 formatted_courses["CourseDescription"] = course.CourseDescription
                 formatted_courses["CourseCredit"] = course.CourseCredit
-
-                course_equivalence_non_oc_data = Course.objects.get(CourseNumber=course.CourseEquivalenceNonOC)
+                try:
```

Comment:
If the OC equivalent Course isn't located in are database yet results while be undefined instead of the website breaking.